windows/python-setup/python-exe.bat

```diff
@@ -3,6 +3,11 @@
 Set version=3.4.3
 Set python32Bit=python-%version%.msi
 Set python64Bit=python-%version%.amd64.msi
+Set pythonPath=%cd:~0,2%/Python34/Scripts
+
+rem Add the pythonPath to environment variables.
+:Add_python_Path
+Setx Path "%pythonPath%;%Path%"

```

Comment:
Oh nice! That would be a huge step forwards.

* How many Windows systems does this work on?
* If Python is already installed (for instance if people are upgrading), will this be skipped?

Reply:
> How many Windows systems does this work on?

It works on Windows 7, 8.1, 10 and Windows Server.

> If Python is already installed (for instance if people are upgrading), will this be skipped?

No, I will add a code to check if the python path exists in the environment variable.